packages/scripts/README.md

```diff
@@ -93,13 +97,13 @@ This is how you execute the script with presented setup:
 
 -   `npm run build` - builds the code for production.
 -   `npm run build:custom` - builds the code for production with two entry points and a custom output directory. Paths for custom entry points are relative to the project root.
--   `npm run build:copy-php` - builds the code for production and opts into copying PHP files from the `src` directory and its subfolders to the output directory.
+-   `npm run build:copy-php` - builds the code for production and opts into copying all PHP files from the `src` directory and its subfolders to the output directory. By default, only PHP files listed in the `render` field in the detected `block.json` files get copied.
```

Comment:
> By default, only PHP files listed in the `render` field in the detected `block.json` files get copied.

As I understand from https://github.com/WordPress/gutenberg/pull/42430 the new `render` field in `block.json` only accepts a path for just one file.

Shouldn't this say something like...
> By default, only the PHP file defined for the `render` field in the detected `block.json` get copied.

Reply:
There might be multiple `block.json` file.

Reply:
In that case, it's perfect in the way it's written 👍